spago.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  dependencies:
+    - console
```

Comment:
Doesn't appear to really depend on `console`.

Reply:
Must've left it there after copying a template from some other package 😆